base-containers/base/bin/_run_student_intern

```diff
@@ -79,17 +111,71 @@ print("Chdir")
 os.chdir(start_cmd["working_dir"])
 print("Popen")
 print(fds)
-p = subprocess.Popen(shlex.split(start_cmd["command"]), preexec_fn=setlimits, stdin=fds[0], stdout=fds[1], stderr=fds[2])
 
-# Handle the signals
-print("Signal thread")
-thread = threading.Thread(target=lambda: handle_signals(p, client), daemon=True)
-thread.start()
+if start_cmd["ssh"]:
```

Comment:
Is it possible for this to be in its own command? It's strange to run `run_student` without cmd and for it to start the ssh server itself.

Three reasons:
1) I think the same could be achieved with `run_student start_the_ssh_server_please`
2) The behavior of run_student would be more consistent
3) It would be nice to parametrize more the way the ssh server works. For exemple, here we always start `bash` when someone connects, but we may want to start other things: `run_student start_the_ssh_server_please --with-shell=zsh`

Reply:
I see a problem with this: it doesn't warn the agent that the student container needs a ssh port. We can keep the ssh flag for `run_student`: `run_student --port=22 start_the_ssh_server_please --with-shell=zsh` or something like that.

This would make the system more versatile too.